service/create/service.go

```diff
@@ -243,7 +239,7 @@ func (s *Service) Boot() {
 						policy = &awsresources.Policy{
 							ClusterID: cluster.Spec.Cluster.Cluster.ID,
 							KMSKeyArn: kmsKey.Arn(),
-							S3Bucket:  s.s3Bucket,
+							S3Bucket:  cluster.Name,
```

Comment:
What?

S3 buckets are flat cannot be nested. You create a bucket with some name and you can put folders inside. And of course, folders can be hierarchical/nested. That said, I don't have a clear picture of what you're proposing.

You want to have a bucket for a customer and folders for clusters inside?